docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.15.0 (Pending)
 ================
+* request_id_extension: fixed static initialization for noop request id extension.
```

Comment:
Can you add in the 1.14.1 release note like you did in the release branch so it matches the release we are about to do? (And then remove from 1.15.0)

Reply:
Ah, now I understand the ordering.. All set!